http.go

```diff
@@ -76,8 +76,13 @@ func probeHTTP(target string, w http.ResponseWriter, module Module) (success boo
 	if config.Method == "" {
 		config.Method = "GET"
 	}
+	if config.Path == "" {
+		config.Path = "/"
+	}
+
+	log.Infof("probeHTTP to %s%s", target, config.Path)
```

Comment:
Where I come from we log all service actions as a record of activity (audit log) and to aid in service analysis.